CS483/CS483/Kartaclysm/Services/IO/InputActionMapping.cpp

```diff
@@ -109,59 +109,70 @@ namespace Kartaclysm
 			return;
 		}
 
-		HeatStroke::Event* pEvent = new HeatStroke::Event("PlayerInput_" + p_iPlayer);
+		// Helpful variables to avoid repetition
+		std::string strPlayerIdentifier = "P" + std::to_string(p_iPlayer) + "_";
```

Comment:
Easy switch. I thought since it was repeated it could be handled just the one time.